openedx/core/djangoapps/credentials/tests/mixins.py

```diff
@@ -38,14 +38,14 @@ class CredentialsDataMixin(object):
                 id=1,
                 username='test',
                 credential=factories.ProgramCredential(
-                    program_id=1
+                    program_uuid='aa7a5470-6962-4ff1-935b-b75f33d977b0'
```

Comment:
I am now passing in data rather than changing these hardcoded values in the mixin